### PR TITLE
RHDEVDOCS-3295 Add a link to the OL 5.0.8 and 5.1.2 advisories to the 5.0 release…

### PR DESCRIPTION
**Pull request by rolfedh**
… notes

- Aligned team: Dev Tools
- For branches: enterprise-4.7 only (NOT main)
- Jira: https://issues.redhat.com/browse/RHDEVDOCS-3295
- Direct link to doc preview: https://deploy-preview-36329--osdocs.netlify.app/openshift-enterprise/latest/logging/cluster-logging-release-notes?utm_source=github&utm_campaign=bot_dp
- SME review: @periklis 
- QE review: @kabirbhartiRH 
- Peer review: @robin-owen
- All reviews complete. Please merge now.

### CODE REVIEW

**Review by periklis — COMMENTED** [review]
LGTM

**Review by periklis — COMMENTED** [review]
This looks like missing the 5.1.2 changelog file?!

**Comment by rolfedh** [comment]
> This looks like missing the 5.1.2 changelog file?!

Yes we're adding it right now.

File: logging/cluster-logging-release-notes.adoc [hunk 1]
```diff
@@ -1,17 +1,46 @@
...
+
+.Supported versions
+
+OpenShift Logging version 5.1 runs on {product-title} versions 4.7 and 4.8.
+
+.Advisories
 
 The following advisories are available for OpenShift Logging 5.1.x:
 
+* link:https://access.redhat.com/errata/RHBA-2021:3545[RHBA-2021:3545 - Bug Fix Advisory. Openshift Logging Bug Fix Release 5.1.2]
```
Comment (rolfedh) [inline logging/cluster-logging-release-notes.adoc:25]: @kabirbhartiRH

File: logging/cluster-logging-release-notes.adoc [hunk 2]
```diff
@@ -1,17 +1,46 @@
...
+= Release notes for Red Hat OpenShift Logging
 include::modules/common-attributes.adoc[]
 :context: cluster-logging-release-notes-v5x
 
 toc::[]
 
-[id="openshift-logging-about-this-release"]
-== About this release
+[id="openshift-logging-inclusive-language"]
+== Making open source more inclusive
```
Comment (robin-owen) [inline logging/cluster-logging-release-notes.adoc:9]: I believe that this section is automatically included in documentation sets published on the Customer Portal based on an internally-available AsciiDoc snippet (unfortunately, I can't remember where it's defined). Including it here could potentially lead to a duplication of the section when published downstream.

As always, feel free to disregard this comment if I'm remembering this incorrectly, of course! :)

**Comment by robin-owen** [comment]
The wording of this PR looks good to me from a documentation perspective. Added a single comment questioning if we should include a copy of the "making open source more inclusive" section, as I believe this section is added automatically and including it in this PR could potentially result in a duplicate section (or, in case of updated wording in the snippet of origin, out of date information).

I'm leaving this review as a comment since I cannot independently confirm whether or not this change is _explicitly_ required. Other than that section, this PR looks good to me! :+1:

File: logging/cluster-logging-release-notes.adoc (hunk 2 above)
Comment (rolfedh) [reply logging/cluster-logging-release-notes.adoc:9]: Thanks for asking, Kevin. I did a quick search and see this statement simply pasted in a variety of locations. In the enterprise-4.* branches, it also appears in the OpenShift release notes.
```
Results for "Making open source more inclusive" in "openshift-docs/*.adoc"

/home/rdlugyhe/openshift-docs/jaeger/rhbjaeger-release-notes.adoc:
  12:  == Making open source more inclusive


/home/rdlugyhe/openshift-docs/logging/cluster-logging-release-notes.adoc:
  14:  == Making open source more inclusive


/home/rdlugyhe/openshift-docs/cicd/gitops/gitops-release-notes.adoc:
  21:  == Making open source more inclusive


/home/rdlugyhe/openshift-docs/cicd/pipelines/op-release-notes.adoc:
  22:  == Making open source more inclusive


/home/rdlugyhe/openshift-docs/service_mesh/v1x/servicemesh-release-notes.adoc:
  8:  == Making open source more inclusive


/home/rdlugyhe/openshift-docs/service_mesh/v2x/servicemesh-release-notes.adoc:
  8:  == Making open source more inclusive

```

Comment (robin-owen) [reply logging/cluster-logging-release-notes.adoc:9]: Having the same text copied and pasted across multiple different documents sounds like a recipe for trouble to me (having dealt with exactly that problem while working across the documentation set for the Go, Rust, and LLVM compilers). It's a maintenance problem, essentially.

By the way, I don't think that problem _needs_ to be addressed **right now** in this pull request. I do think that another issue should be opened at some point to discuss how this information can be presented in a more flexible and maintainable way, however. (Personally, I'd advocate for the notice itself being a reusable module or "snippet" if those are in use for OpenShift docs.)

Just something to think about! :)

Comment (rolfedh) [reply logging/cluster-logging-release-notes.adoc:9]: Good suggestion! I've created [RHDEVDOCS-3320 Create a "Making open source more inclusive" module for reuse](https://issues.redhat.com/browse/RHDEVDOCS-3320)

File: modules/cluster-logging-release-notes-5.0.8.adoc [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+= OpenShift Logging 5.0.8
+
+This release includes link:https://access.redhat.com/errata/RHBA-2021:3526[RHBA-2021:3526 - Bug Fix Advisory. OpenShift Logging Bug Fix Release (5.0.8)].
+
+[id="openshift-logging-5-0-8-bug-fixes"]
+== Bug fixes
+
+This release also includes the following bug fixes:
+
+* link:https://issues.redhat.com/browse/LOG-1741[LOG-1741] - ElasticSearch Operator isn't upgraded due to wrong olm.skipRange
```
Comment (Preeticp) [inline modules/cluster-logging-release-notes-5.0.8.adoc:11]: We need to elaborate this bug fix to match the OCP descriptions for bug fixes: https://docs.openshift.com/container-platform/4.8/release_notes/ocp-4-8-release-notes.html#ocp-4-8-bug-fixes and use the RN text provided by the Devs in the Jira: Due to an issue in the release pipeline scripts, the value of the `olm.skipRange` field remained unchanged at `5.2.0` and was not updated when the z-stream number, `0`, increased. The current release fixes the pipeline scripts to update the value of this field when the release numbers change.

**Comment by Preeticp** [comment]
@rolfedh apart from the one comment, looks good to me. Let me know when it is ready to be merged.

**Pull request merged**